DESCRIPTION.md

```diff
@@ -8,6 +8,10 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
+- v3.12.0(TBD)
+  - Optimized `to_pandas()` performance by fully parallel downloading logic.
```

Comment:
do we need this change for sproc connector?

Reply:
We will want it, do you who is the best contact to talk about this?

Reply:
check with @sfc-gh-zyao , it used to be us initiating the PR but I don't know the current procedure.